### PR TITLE
Simplify opfab configuration

### PR DESCRIPTION
**Pull request by freddidierRTE**
- In release note :
  -  In chapter : Tasks
  -  Text : #4687 Simplify configuration

### CODE REVIEW

File: config/dev/businessconfig-dev.yml [hunk 1]
```diff
@@ -1,8 +1,4 @@
-server:
-  port: 2100
-spring:
-  application:
-    name: businessconfig
+# This file contains only parameters that needs to be override compare to docker configuration
```
Comment (quinarygio) [inline config/dev/businessconfig-dev.yml:1]: to be overridden
or
to be overwritten

File: config/dev/cards-consultation-dev.yml [hunk 2]
```diff
@@ -1,9 +1,3 @@
-server:
-  port: 2104
-spring:
-  application:
-    name: cards-consultation
-#logging.level.org.opfab: debug
+# This file contains only parameters that needs to be override compare to docker configuration
```
Comment (quinarygio) [inline config/dev/cards-consultation-dev.yml:1]: same as before

File: config/dev/cards-publication-dev.yml [hunk 3]
```diff
@@ -1,33 +1,4 @@
...
-  kafka:
-    topics:
-      card:
-        topicname: opfab
-      response-card:
-        topicname: opfab-response
-    schema:
-      registry:
-        url: http://localhost:8081
+# This file contains only parameters that needs to be override compare to docker configuration
```
Comment (quinarygio) [inline config/dev/cards-publication-dev.yml:1]: same as before

File: config/dev/common-dev.yml [hunk 4]
```diff
@@ -1,63 +1,19 @@
-server:
-  forward-headers-strategy: framework
+# This file contains only parameters that needs to be override compare to docker configuration
```
Comment (quinarygio) [inline config/dev/common-dev.yml:1]: same as before

**Review by quinarygio — CHANGES_REQUESTED** [review]

File: config/dev/cards-publication-dev.yml (hunk 3 above)
Comment (vlo-rte) [inline config/dev/cards-publication-dev.yml:1]: that needs => that need
compare to docker configuration => from the docker configuration

File: config/dev/common-dev.yml (hunk 4 above)
Comment (vlo-rte) [inline config/dev/common-dev.yml:1]: that needs => that need
compare to docker configuration => from the docker configuration

File: src/docs/asciidoc/deployment/configuration/configuration.adoc [hunk 5]
```diff
@@ -14,78 +14,55 @@
...
 
 |===
-|name|default|mandatory?|Description
+|name|default|Description
 
 
-|operatorfabric.servicesUrls.users||yes|Indicates where the Users service can we reached (to get information about the current user).
-|operatorfabric.userActionLogActivated|yes|no|Indicates if the user action log feature is enabled or not.
-
+|operatorfabric.servicesUrls.users|users:2103|Indicates where the Users service can be reached form the other services.
```
Comment (vlo-rte) [inline src/docs/asciidoc/deployment/configuration/configuration.adoc:47]: form => from

File: src/docs/asciidoc/deployment/configuration/configuration.adoc [hunk 6]
```diff
@@ -14,78 +14,55 @@
...
+|operatorfabric.servicesUrls.businessconfig|businessconfig:2100|Indicates where the Business service can be reached form the other services.
```
Comment (vlo-rte) [inline src/docs/asciidoc/deployment/configuration/configuration.adoc:48]: form => from

File: src/docs/asciidoc/deployment/configuration/configuration.adoc [hunk 7]
```diff
@@ -426,4 +301,84 @@ spring:
...
+|cardsExternalDiffusion.opfab.password||yes|user password to call Opfab services
+|cardsExternalDiffusion.defaultConfig.mailFrom||yes|Mail from address
+|cardsExternalDiffusion.defaultConfig.subjectPrefix||yes|Mail subject prefix
+|cardsExternalDiffusion.defaultConfig.bodyPrefix|||Mail body prefix
+|cardsExternalDiffusion.defaultConfig.windowInSecondsForCardSearch|||Max time interval used to query new cards (in seconds)
+|cardsExternalDiffusion.defaultConfig.secondsAfterPublicationToConsiderCardAsNotRead|||Time period after card publication before sending mail notification
+|cardsExternalDiffusion.defaultConfig.checkPeriodInSeconds|||Time interval between consecutive checks
+|cardsReminder.adminPort||yes|Cards reminder service Listen port
+|cardsReminder.activeOnStartup||yes|Flag to start the cards reminder service on startup
+|cardsReminder.checkPeriodInSeconds|||Cards remonder time interval between consecutive checks
```
Comment (vlo-rte) [inline src/docs/asciidoc/deployment/configuration/configuration.adoc:372]: remonder => reminder

File: src/docs/asciidoc/deployment/configuration/configuration.adoc [hunk 8]
```diff
@@ -426,4 +301,84 @@ spring:
...
+
+
+==== Cards-consultation service
+
+The cards-consultation service has these specific properties :
+
+|===
+|name|default|Description
+
+|checkIfUserIsAlreadyConnected|true|If false, OperatorFabric will allow a user to have several sessions opened at the same time. However, it may cause synchronization problems between the sessions using the same login, so it is recommended to let it to true, its default value.
```
Comment (vlo-rte) [inline src/docs/asciidoc/deployment/configuration/configuration.adoc:314]: let it to true => let it true

**Review by vlo-rte — CHANGES_REQUESTED** [review]
See my comments please.

**Review by vlo-rte — APPROVED** [review]

**Pull request merged**